Jenkinsfile

```diff
@@ -25,6 +25,12 @@ dockerNode(image: 'uf-mil:subjugator') {
 	}
 	stage("Format") {
 		sh '''
+			if [[ ! -z "$(python2.7 -m flake8 --max-line-length=120 --exclude=__init__.py .)" ]]; then
```

Comment:
All this rule appears to do is prevent you from defining a lambda, which is not required to use one as a callback. I think it is right in that lambdas should be ephemeral.